### PR TITLE
Update use_cases.md

### PR DESCRIPTION
**Pull request by iskyOS**
Updates the use cases docs page for new commands function plus inline chat for refactors.

## Test plan

Docs only.

### CODE REVIEW

File: doc/cody/explanations/use_cases.md [hunk 1]
```diff
@@ -20,30 +20,25 @@ For example:
...
 - "Write a new graphql resolver in Go for a new object type called PitaOrders"
 
-### Write code with fixup
+### Refactor code with inline chat
 
-Cody can also add and edit code directly in your file rather than making suggestions. This functionality is called "fixup," and it's designed to keep you in flow writing code alongside Cody.
+Cody can also edit code directly in your file rather than making suggestions. You can prompt Cody to refactor code using inline chat.
 
-To fixup some code, try:
+To refactor code inline, try:
```
Comment (MaedahBatool) [inline doc/cody/explanations/use_cases.md:27]:
```suggestion
To refactor code inline:
```

File: doc/cody/explanations/use_cases.md [hunk 2]
```diff
@@ -20,30 +20,25 @@ For example:
...
 
-1. Write something you'd like Cody to do inline in your file
-2. Highlight this code plus any related code you'd like changed, then use the `ctrl`+`alt`+`/` hotkey (or the `Cody: Fixup` VS Code command)
-3. Cody will work directly in your file
+1. Highlight the snippet of code you'd like to refactor.
+2. Click the `+` icon in the gutter to the left of the first line of code.
```
Comment (MaedahBatool) [inline doc/cody/explanations/use_cases.md:30]:
```suggestion
2. Click the `+` icon to the left of the first line of code.
```

File: doc/cody/explanations/use_cases.md [hunk 3]
```diff
@@ -20,30 +20,25 @@ For example:
...
+3. In the Cody chat box, type `/fix` + your prompt for Cody. For example: `/fix refactor this code to be more easily readable`.
+4. Click `Ask Cody`.
```
Comment (MaedahBatool) [inline doc/cody/explanations/use_cases.md:32]:
```suggestion
4. Click `Ask Cody` (or alternatively use shortcut `Cmd + Enter`)
```

**Review by MaedahBatool — CHANGES_REQUESTED** [review]
Overall looks good. Left a few comments. :)

**Comment by iskyOS** [comment]
Thanks @MaedahBatool! I've updated. I also made one more commit to clean up formatting and make a note about inline chat being VS Code only at the moment.

**Pull request merged**